dotnet/src/Connectors/Connectors.AzureOpenAI.UnitTests/KernelCore/KernelTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Moq;
+
+namespace SemanticKernel.Connectors.AzureOpenAI.UnitTests.KernelCore;
+
+public sealed class KernelTests : IDisposable
+{
+    private readonly MultipleHttpMessageHandlerStub _multiMessageHandlerStub;
+    private readonly HttpClient _httpClient;
+    private readonly Mock<ILoggerFactory> _mockLoggerFactory;
+    private readonly Mock<ILogger<KernelTests>> _mockLogger;
+
+    public KernelTests()
```

Comment:
nit: Not sure about `KernelCore/KernelTests` naming here. I would put it in already existing `Services` folder with `AzureOpenAITelemetryTests` name. But it's a personal preference, feel free to skip. Same applies for OpenAI connector.